.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
I believe the .idea files are from intelliJ or another IDE and shouldn't be included, there is a change in the other PR for adding these to the .gitignore